test/support/stores/rest-array.rb

```diff
@@ -27,7 +27,7 @@ def remove_from_scope(factory_name, class_name, list_of_modules, config)
       visit('/admin')
       select_last_module_from_list(list_of_modules)
       assert_not page.has_content?('New sports diet')
-      assert page.has_css?('div.item-title', count: count_elements_in_scope_before-1)
+      assert page.has_css?('.item.is-object', count: count_elements_in_scope_before-1)
```

Comment:
Line is too long. [86/80]<br>Surrounding space missing for operator `-`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.